http_endpoints.go

```diff
@@ -8,15 +8,17 @@ import (
 	"strings"
 
 	devcycle "github.com/devcyclehq/go-server-sdk/v2"
+	devcycle_api "github.com/devcyclehq/go-server-sdk/v2/api"
 	"github.com/gin-gonic/gin"
 )
 
 func Health(c *gin.Context) {
 	c.Status(200)
 }
 
-func Variable(client *devcycle.Client) gin.HandlerFunc {
+func Variable() gin.HandlerFunc {
 	return func(c *gin.Context) {
+		client := c.Value("devcycle").(*devcycle.Client)
```

Comment:
ohh, interesting so the devcycle client is now attached to the request context?

Reply:
Yeah - I wanted to make it more context friendly and properly scoped to each request since it's handled by a different gin instance